tests/TestCase/Cache/Engine/XcacheEngineTest.php

```diff
@@ -75,7 +75,7 @@ public function tearDown()
      */
     public function testConfig()
     {
-        $config = Cache::engine('xcache')->config();
+        $config = Cache::getConfig('xcache')->getConfig();
```

Comment:
`Cache::engine()` did not deprecate 
I think your want to use 
`Cache::engine('xcache')->getConfig();`

Reply:
Derp. You're right this is a mistake.